eventhubs/client/README.md

```diff
@@ -229,41 +241,50 @@ client.close();
 ## Troubleshooting
 
 ### Enable client logging
+
 You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For example,
 setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can be found 
 here: [log levels][log_levels].
 
 ### Enable AMQP transport logging
+
 If enabling client logging is not enough to diagnose your issues. You can enable logging to a file 
 in the underlying AMQP library, [Qpid Proton-J][qpid_proton_j_apache]. Qpid Proton-J uses `java.util.logging`. You can 
 enable logging by create a configuration file with the contents below. Or set `proton.trace.level=ALL` and whichever 
 configuration options you want for the `java.util.logging.Handler` implementation. Implementation classes and their
 options can be found in [Java 8 SDK javadoc][java_8_sdk_javadocs].
 
-#### Sample "logging.config" file ####
-``` logging config
+#### Sample "logging.config" file
+
+The configuration file below logs trace output from proton-j to the file "proton-trace.log".
+
+``` loggingConfig
```

Comment:
Remove loggingConfig. It's not a language that is understood by their syntax highlighting. https://github.com/github/linguist/blob/master/lib/linguist/languages.yml these are the supported languages.